policyengine_us/tests/policy/baseline/gov/states/nj/tax/income/credits/nj_child_tax_credit/nj_child_tax_credit.yaml

```
@@ -0,0 +1,141 @@
+- name: Filler with 2 dependents under 6 and taxable income under 30000 gets 1000 credit when not filing separately
```

Please also apply to other tests
```suggestion
- name: Filer with 2 dependents under 6 and taxable income under $30,000 gets $1,000 credit when not filing separately
```